server/index.ts

```diff
@@ -12,17 +12,19 @@ const { t } = useTranslate('en');
 type AccountData = { token: string } & Account;
 
 const loginCallbacks: Array<(player: alt.Player) => void> = [];
+const loggedInPlayers = [];
 const sessionKey = 'can-authenticate';
 const db = Rebar.database.useDatabase();
 
 function setAccount(player: alt.Player, account: Account) {
+    const logPlayer = { _id: account._id };
     Rebar.document.account.useAccountBinder(player).bind(account);
     Rebar.player.useWebview(player).hide('Auth');
     Rebar.player.useNative(player).invoke('triggerScreenblurFadeOut', 1000);
     player.deleteMeta(sessionKey);
     player.dimension = 0;
     player.emit(AuthEvents.toClient.cameraDestroy);
-
+    loggedInPlayers.push(logPlayer);
```

Comment:
I think a better way to store this is using a map.

```ts
loggedInPlayers[player.id] = account._id;
```

Then you can iterate over Object valuesuntil you find a matching account_id.

You could use player disconnect event to remove the entry from the map.

Reply:
Fixed with the commit 52fbcaa.